contracts/proxy/VolatilePoolProxy.sol

```diff
@@ -52,90 +59,140 @@ contract VolatilePoolProxy is Initializable, OwnableUpgradeSafe, Helper {
 
     event Initialized(address indexed thisAddress);
 
-    function initialize() public initializer {
+    function initialize(address _advisor) public initializer {
         OwnableUpgradeSafe.__Ownable_init();
+        OwnableUpgradeSafe.transferOwnership(_advisor);
```

Comment:
Same as above, typically the initializer gets ownership

Reply:
@herman-rogers I am sure about this that's why the function transferownership lets keep it on a good to have test for now